e2e/backend_service_update_test.go

```diff
@@ -226,3 +255,86 @@ func TestBackendServiceUpdate(t *testing.T) {
 	wantBS = &compute.BackendService{LoadBalancingScheme: "INTERNAL_MANAGED"}
 	checkBackendService(t, ctx, theCloud, bs1ID, wantBS, compareLBScheme)
 }
+
+func TestBackendServiceMetadataUpdate(t *testing.T) {
+	t.Parallel()
+
+	ctx := context.Background()
+	graphBuilder := rgraph.NewBuilder()
+	resUniqueIdPart := "test-bs-metadata-update"
+
+	hcName := resourceName(resUniqueIdPart + "-hc")
+	hcID, err := buildHealthCheck(graphBuilder, hcName, 15)
```

Comment:
I don't think that HC is needed in this test, it will just make test execute longer but it has no meaning to the logic.

Reply:
Done.